x-pack/plugins/observability_solution/apm/public/components/shared/suggestions_select/index.tsx

```diff
@@ -52,6 +52,9 @@ export function SuggestionsSelect({
 
   const { data, status } = useFetcher(
     (callApmApi) => {
+      if (!searchValue) {
```

Comment:
Now we don't have extra API calls but the problem is when the user clicks on the combo box there are no suggestions anymore (only if the input text is changed we see the suggestions). In the issue I see that it's expected to also load the options when the field is initialized (so when the box is clicked if I understand that correctly): 

> The request for suggestions should only occur when the input field is modified or **initialized**,

I am adding a screen recording to show this issue (left is the current PR change, the right is how it worked before)


https://github.com/user-attachments/assets/cb6e2769-3ce2-415d-943b-148382baeb3e